hooks/useValidatedInput.js

```diff
@@ -1,105 +0,0 @@
-import { useState, useCallback, useMemo } from 'react';
```

Comment:
Removed this because it is more than we need and its use of floats causes precision loss